.golangci.yml

```diff
@@ -34,24 +34,25 @@ linters-settings:
     local-prefixes: kubevirt.io/kubevirt
   mnd:
     # don't include the "operation" and "assign"
-    checks: argument,case,condition,return
+    checks:
+      - argument
+      - case
+      - condition
+      - return
     ignored-functions:
-      - '^Eventually$'
-      - '^EventuallyWithOffset$'
-      - '^ExpectWithOffset$'
+      - "^Eventually$"
```

Comment:
The mnd [example](https://golangci-lint.run/usage/linters/#mnd) uses single quotes, could you please elaborate why is there a need to change them?

Reply:
Apologies, as discussed I found most of these issues after switching back to `nvim`. I thought this was coming from `yamlint` via `nvim-lint` as with the other warnings/errors but it's actually coming from `yaml-language-server` via `nvim-lspconfig` that defaults to formatting YAML with single quotes. I've disabled this now and will back out these changes.